cypress/e2e/blueprints/explorer/cluster/events.js

```diff
@@ -0,0 +1,3913 @@
+// GET /v1/events - small set of events data
+const eventsGetResponseSmallSet = {
+  type:         'collection',
+  links:        { self: 'https://yonasb29h.qa.rancher.space/v1/events' },
```

Comment:
Does the URL matter for this data to be visible in the UI?

Reply:
updated to a random prefix. still works the same